### PR TITLE
Update routing.rst

### PR DESCRIPTION
**Pull request by damianoporta**

### CODE REVIEW

File: en/development/routing.rst [hunk 1]
```diff
@@ -580,10 +580,10 @@ By using extensions, you tell the router to remove any matching file extensions,
 and then parse what remains. If you want to create a URL such as
 /page/title-of-page.html you would create your route using::
 
-    Router::scope('/api', function ($routes) {
-        $routes->extensions(['json', 'xml']);
+    Router::scope('/page', function ($routes) {
+        $routes->extensions(['json', 'xml', 'html']);
```
Comment (markstory) [inline en/development/routing.rst]: I don't think you need to add the html extension.

Comment (damianoporta) [inline en/development/routing.rst]: ok!

Comment (cake17) [inline en/development/routing.rst]: @damianoporta is this possible for you to remove `html` in the array ?

**Comment by cake17** [comment]
I'll merge this and make the edits in another commit

**Comment by cake17** [comment]
thanks @damianoporta

**Pull request merged**